### ISSUE TITLE
76,5% Accuracy on LFW

### ISSUE THREAD

**Issue by Jusya** [davidsandberg/facenet#42]
Hello, 
I have low accuracy on LFW using this FaceNet implementation. I'm using "All images as gzipped tar file" - the first offered dataset from list of downloadable datasets on http://vis-www.cs.umass.edu/lfw/
Photo attached:
![lfw_datasets](https://cloud.githubusercontent.com/assets/13995949/18201445/390008cc-711b-11e6-8029-c2d8cb6e91a9.jpg)
Am I doing something wrong? Which dataset should I use to check accuracy and get 0.919+-0.008?
It would be great if you could help we with that issue.
Thank you.

**Comment by pby5** [comment]
training dataset and testing dataset should use same face alignment so LFW dataset and training dataset should use same face alignment and I reproduce the same accurary as the author did

**Comment by Jusya** [comment]
I understand that) I thought, that the pre-trained model "model-20160506.ckpt-500000" which I use should give 0.919+-0.008 accuracy on LFW as it was mentioned in Performance in README file. However, I got 76,5% accuracy.
If pre-trained model was trained on a combination of FaceScrub and CASIA-Webface what should I do with LFW to get high accuracy? Should I download another dataset to check performance, or should I somehow change (e.g. change face alignment) LFW dataset that I've downloaded by myself or with function that is already written in this FaceNet implementation?

Thank you for answering me)

**Comment by davidsandberg** [comment]
Yes, you need to align the images in the LFW dataset as well.
On the [wiki](https://github.com/davidsandberg/facenet/wiki/Validate-on-lfw) there's a description of which LFW files to donwload and how to run the alignment and evaluation.
A tricky thing is that dlib does not manage to find faces in all the images, so for those images the deep funneled version of the LFW dataset is used. But it's all described on the wiki.

**Comment by Jusya** [comment]
Thank you for you help! I will go through the wiki tutorial you've provided)
